### PR TITLE
Front end 3

### PR DESCRIPTION
**Pull request by davidklaver**
Addressed following features/issues from most recent meeting with Nancy:
- Spousal refusal input should only appear if it's a spousal case
- Notes textarea should be wider
- Add column in not tracking for pickup date, notes

@waltertschwe could you please review this and push it up to QA?

### CODE REVIEW

**Comment by waltertschwe** [comment]
This PR is pretty much unreviewable and not mergeable lol.  I think the symptom is the current state of how Medicaid is setup because I don't think your wrote 5k+ lines of code and deleted 1k.   Will need to spend some time reviewing how Medicaid works.

File: app/settings.py [hunk 1]
```diff
@@ -82,6 +82,10 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'medicaid',
+        # 'USER': 'marc',
```
Comment (waltertschwe) [inline app/settings.py]: this merge will break QA.  Need to have the dynamic database settings.

Comment (davidklaver) [reply app/settings.py]: I assume you mean by using env variables?

Comment (davidklaver) [reply app/settings.py]: Like so?
```suggestion
        'USER': os.getenv('POSTGRES_USERNAME'),
        'PASSWORD': os.getenv('POSTGRES_PASSWORD'),
```

**Pull request merged**